lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -223,5 +223,13 @@ def requires_7_day_notice?
     def same_sex_alt_fees_table_country?
       @data_query.ss_alt_fees_table_country?(ceremony_country, self)
     end
+
+    def outcome_path_when_resident_in_uk
+      [
+        '', 'marriage-abroad', 'y',
+        @ceremony_country, 'uk',
+        @partner_nationality, @sex_of_your_partner
+      ].join('/')
+    end
```

Comment:
Oh. I've just noticed that `current_path` isn't a method on `State`.

I suppose I'm wary of the idea that the path to a question or outcome is built up of a set of responses separated by slashes being spread further around the codebase. But perhaps that was already a problem in the `current_path` precalculate block.